### PR TITLE
porting some information from kubeadm-init.md to kubelet-integration.md

### PR DESCRIPTION
**Pull request by vorburger**
from reference/setup-tools/kubeadm/kubeadm-init.md to setup/production-environment/tools/kubeadm/kubelet-integration.md

includes edits I just proposed in #15421 and https://github.com/kubernetes/kubernetes/pull/80136

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 1]
```diff
@@ -157,15 +157,19 @@ has finished performing the TLS Bootstrap.
 
 ##  The kubelet drop-in file for systemd
 
-The configuration file installed by the kubeadm DEB or RPM package is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+`kubeadm` ships with configuration for how the `kubelet` should
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md]: How about: “for how systemd should run kubelet”.

Comment (vorburger) [reply content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md]: Tx; done. PS: Pro Tip, use GitHub's cool [suggest a specific change](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) (Pt. 6) feature!

File: content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 2]
```diff
@@ -157,15 +157,19 @@ has finished performing the TLS Bootstrap.
 
 ##  The kubelet drop-in file for systemd
 
-The configuration file installed by the kubeadm DEB or RPM package is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+`kubeadm` ships with configuration for how `systemd` should run `kubelet`.
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md]: please remove quotes around systemd, kubeadm and kubelet.

> kubeadm ships with configuration for how systemd should run the kubelet

File: content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 3]
```diff
@@ -157,15 +157,19 @@ has finished performing the TLS Bootstrap.
 
 ##  The kubelet drop-in file for systemd
 
-The configuration file installed by the kubeadm DEB or RPM package is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+`kubeadm` ships with configuration for how `systemd` should run `kubelet`. 
+Note that the `kubeadm` CLI command never touches this drop-in file.
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md]: remove quotes in the new changes, please.

**Comment by vorburger** [comment]
@neolit123 Please use GitHub's cool [suggest a specific change](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) (Pt. 6) feature - instead of describing the change requested in words, just click that Icon before the aA B which let's you kind of "inline edit" and counter propose. It's easier because faster for both the reviewer (you) and the Pull Request author (me) to "parse" and easily accept.

**Comment by neolit123** [comment]
@vorburger i know how the feature works. :) i'm leaving the edits to you.

**Comment by vorburger** [comment]
> @vorburger i know how the feature works. :) i'm leaving the edits to you.

done. LGTY now?

File: content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 4]
```diff
@@ -157,15 +157,19 @@ has finished performing the TLS Bootstrap.
 
 ##  The kubelet drop-in file for systemd
 
-The configuration file installed by the kubeadm DEB or RPM package is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+kubeadm ships with configuration for how systemd should run the kubelet. 
+Note that the kubeadm CLI command never touches this drop-in file.
+
+This configuration file installed by the `kubeadm` [DEB](https://github.com/kubernetes/kubernetes/blob/master/build/debs/10-kubeadm.conf) or [RPM package](https://github.com/kubernetes/kubernetes/blob/master/build/rpms/10-kubeadm.conf) is written to
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md]:
```suggestion
This configuration file installed by the kubeadm [DEB](https://github.com/kubernetes/kubernetes/blob/master/build/debs/10-kubeadm.conf) or [RPM package](https://github.com/kubernetes/kubernetes/blob/master/build/rpms/10-kubeadm.conf) is written to
```

File: content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 5]
```diff
@@ -157,15 +157,19 @@ has finished performing the TLS Bootstrap.
...
+`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by `systemd`.
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md]:
```suggestion
`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
```

**Comment by vorburger** [comment]
Can I do anything else to help finally get this merged?

/assign @tfogo

**Comment by vorburger** [comment]
Can I do anything else to help get this merged?

**Comment by zacharysarah** [comment]
Leaving final `/lgtm` to @neolit123. 👋 

/approve

**Pull request merged**